srcpkgs/NetworkManager-fortisslvpn/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'NetworkManager-fortisslvpn'
+pkgname=NetworkManager-fortisslvpn
+version=1.3.90
+revision=1
+build_style=gnu-configure
+_pppver=2.4.9
+configure_args="--disable-static --libexecdir=/usr/lib/networkmanager --with-pppd-plugin-dir=/usr/lib/pppd/$_pppver"
+hostmakedepends="automake gettext-devel glib-devel intltool libtool pkg-config"
+makedepends="ppp-devel gtk+3-devel libglib-devel libnma-devel libsecret-devel"
+depends="openfortivpn"
```

Comment:
```suggestion
depends="openfortivpn ppp"
```

Reply:
Is this required? I thought that `makedepends="ppp-devel` already makes the package depend on `ppp`...